substrate/frame/child-bounties/src/benchmarking.rs

```diff
@@ -114,7 +119,8 @@ fn activate_bounty<T: Config>(
 	let approve_origin =
 		T::SpendOrigin::try_successful_origin().map_err(|_| BenchmarkError::Weightless)?;
 	Bounties::<T>::approve_bounty(approve_origin, child_bounty_setup.bounty_id)?;
-	Treasury::<T>::on_initialize(BlockNumberFor::<T>::zero());
```

Comment:
we shouldnt allow `on_initialize(0)`: https://github.com/paritytech/polkadot-sdk/issues/4316